packages/lodestar-db/src/schema.ts

```diff
@@ -32,9 +32,15 @@ export enum Bucket {
   proposerSlashing = 14, // ValidatorIndex -> ProposerSlashing
   attesterSlashing = 15, // Root -> AttesterSlashing
   // validator
-  validator = 16,
-  lastProposedBlock = 17,
-  proposedAttestations = 18,
+  // validator = 16, // DEPRECATED on v0.11.0
+  // lastProposedBlock = 17, // DEPRECATED on v0.11.0
+  // proposedAttestations = 18, // DEPRECATED on v0.11.0
+  // validator slashing protection
+  slashingProtectionBlockBySlot = 20,
+  slashingProtectionAttestationByTarget = 21,
+  slashingProtectionAttestationLowerBound = 22,
+  slashingProtectionMinSpanDistance = 23,
+  slashingProtectionMaxSpanDistance = 24,
```

Comment:
nitpick and probably for separate PR: we should probably split beacon node db from validator db. It made sense to be together earlier because of how dependencies were arranged

Reply:
It's indeed confusing. The advantage of using the same schema is that if you re-use db instances there won't be conflicts.